dap-mouse.el

```diff
@@ -166,6 +167,8 @@ If there is an active selection - return it."
           (setq dap-mouse--hide-timer nil)
           (remove-hook 'post-command-hook #'dap-tooltip-post-tooltip)))))))
 
+(declare-function #'lsp-treemacs-render "ext:lsp-treemacs")
```

Comment:
lsp-treemacs is already a dependency, so no need from declare-function just require it.